TROUBLESHOOTING.md

```diff
@@ -1,5 +1,65 @@
 ### Hydrogen installation fails reporting an error running `node-gyp`
 
+**For Windows (Tested with Windows 10)**
+If you have a `node-gyp` error, before you read through the troubleshooting
+section below, I suggest you repeat my steps here as some of the information
+in the following troubleshooting section may be out of date.
+
+- Install Git
+- Install Python 2 (or Anaconda 2) and make it your default python. I tried doing
+  this in virtual environment, but it didn't end up well. Personally, I don't
+  recommend having Python 3 as default and trying to `apm config set python` later
+  to Python 2 as it messed with my `apm -v`, causing python not detected any
+  more.
+- Later if you want to run Python 3 in virtual environment, you can do:
+  ***for Python installer user:***
+  `virtualenv -p <path/to/your/python3> <envname>`
+  ***for Anaconda user:***
+  `conda create -n <envname> python=3.5 anaconda`
+- Install [ZeroMQ](http://zeromq.org/distro:microsoft-windows)
```

Comment:
Well, there are lots of alternatives... VS2015 and Python 2 may not be necessary either, both C++ components and Python 2 are included in `npm install --global --production windows-build-tools`. 

Also after trying it myself and talking to n-riesco, now I prefer installing python 3 and just set npm config and apm config python to my python 2 directory.

Just tell me how you want me to change it. Thanks.